Text for Pillar 3 (#19)
the-teachingRSE-project/institutionalized_education: institutionalised_education.md

Naturally, other existing structures outside of academia mentioned before like the Carpentries [@Carpentries] can be nicely integrated
and offered to a broad audience on a regular basis.

# Which impact can dedicated RSE teaching Institutions have?

> ## The local impact of an RSE institution e.g into a univers

Reviewer: ```suggestion
## The local impact of an RSE institution e.g on a university
```
Also consider deleting the e.g. part, since that seems awkward in a heading to me

The benefits to a university are numerous. Having proper RSE students will improve the digital literacy in the domains.
Local RSEs can reach out into the domain sciences and drive the transition to digital science.
A local RSE teaching institutions can together with other institutions host regular events
like maintainathons, and reproducibility hackathons. The RSE institutions form a natural point
where cross-domain fertilization can occur. And, importantly, adopting RSE institutions early on,
can enable universities to distinguish themselves from the competition.

> Regular training session raise the level of proficiency with the use of digital tools in the domains,

Reviewer: ```suggestion
Regular training sessions raise the level of proficiency with the use of digital tools in the domains,
```

covering e.g. topics like CI or CD.

> Exisiting trans-national entities like the carpentries or the NHR Training-Academy can be easily 

Reviewer: ```suggestion
Exisiting trans-national entities like The Carpentries or the NHR Training-Academy can be easily 
```
